services/app-api/handlers/reports/create.ts

```diff
@@ -56,3 +87,21 @@ export const createReport = handler(async (event, _context) => {
     };
   } else throw new Error(error.MISSING_DATA);
 });
+
+const putObjectWrapper = (
+  s3: S3,
+  params: { Bucket: string; Key: string; Body: string; ContentType: string }
+) => {
+  return new Promise((resolve, reject) => {
+    s3.putObject(params, function (err: any, result: any) {
+      if (err) {
+        // console.log("Put Error", err);
+        reject(err);
+      }
+      if (result) {
+        // console.log("Put Result", result);
+        resolve(result);
+      }
+    });
+  });
+};
```

Comment:
We should pull this out into a separate file as a shared utility instead of writing it in both files.